qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -163,6 +168,7 @@ def getNodeItemForNode(self, node, highlight_level):
                              label=name,
                              shape=node.attr.get('shape', 'ellipse'),
                              color=color,
+                             tooltip=tooltip
```

Comment:
This and the above could be shortened to:

    tooltip=node.attr.get('tooltip', None)